### DISCUSSION TITLE
the n-mod compilation

### DISCUSSION THREAD

**Discussion by creator1116** [landgreen/n-gon#348] [Ideas]
as more and more are created i want to make a list or compilation of n-mods.
also what would be cool: DLCs for n-gon they could be seasonal or smth, unofficial, ofc.
https://github.com/creator1116/the-n-mod-compilation-

**Comment by corneeme** [comment]
sweet! nice to finally see some use out  of the discussions tab

**Comment by tbxyd** [comment]
Yeah, better than issues lol. This past week has had a large spark in the ngon modding community


Sent from my Apple Watch

**Comment by corneeme** [comment]
ikr?!

**Comment by tbxyd** [comment]
Its nice tho. Plus regular ppl trying to play mods would find it hard without a good community


Sent from my Apple Watch

**Comment by corneeme** [comment]
it is

**Comment by 3xionDev** [comment]
> Yeah, better than issues lol. This past week has had a large spark in the ngon modding community

I take full credit for this spike in modding /j

**Comment by tbxyd** [comment]
And credit is def given. All from a radiation issue lol


Sent from my Apple Watch

**Comment by AnnonymousNerd87** [comment]
Nolan you should have used the tone indicator /hj because it is the worst
one/hj

**Comment by 3xionDev** [comment]
> Nolan you should have used the tone indicator /hj because it is the worst
one/hj

I have no idea what you're talking about

**Comment by tbxyd** [comment]
Fr what is tone indicator


Sent from my Apple Watch

**Comment by AnnonymousNerd87** [comment]
tone indicator is /j and /hj, it replaces things like a sarcastic tone.
also asterisks and italics to an extent.

ex. I have $15 and the fate of the world riding on this election /hj

**Comment by AnnonymousNerd87** [comment]
or something like you pretending to do something, like

*Slaps you around a bit with an oversized trout*

it started in ircs but has continued through message boards into the full
internet

in landgreen tradition: https://en.wikipedia.org/wiki/Tone_indicator

**Comment by corneeme** [comment]
lol i will def  start using that

**Comment by AnnonymousNerd87** [comment]
yeah ur welcome /js /hj

**Comment by tbxyd** [comment]
Def gonna start using this too


Sent from my Apple Watch

**Comment by tbxyd** [comment]
Guys i need help naming my mod, any ideas


Sent from my Apple Watch

**Comment by corneeme** [comment]
whats it about?
on an unrelated note, got  the  links from the community links issue, felt like they should be here bc they are all essentially modded ngon
https://ngon.fandom.com/wiki/N-gon
https://3xiondev.github.io/n-gon-upgraded
https://github.com/kgurchiek/n-gon-mobile
https://kgurchiek.github.io/n-gon-portal-gun
https://github.com/kgurchiek/n-gon-controller
https://github.com/kgurchiek/n-gon-multiplayer
https://www.cornbread2100.com/n-gon-loader

**Comment by tbxyd** [comment]
Not really about anything, just stuff that i think is cool and that i added. Check out the repo for details (readme doesn't have full changelog). So far I've added a sniper, some tech for it, a shotgun tech, and a new field. Link for the game is c-rxxp-y.github.io/n-gon-mods


Sent from my Apple Watch

**Reply by smallbigman** [reply]
I love the mod and the combination with perfect diamagnetism and standing wave but the problem is that you can still get fieldtech from perfect diamagnetism and you should code it so you can get multiple of supplementary angles to probably 5x or 6x like diffraction grating
![Screenshot 2024-11-05 4 25 26 PM](https://github.com/user-attachments/assets/2b5ab0e8-4a6d-4c49-9896-23c467927ca7)

**Comment by tbxyd** [comment]
Like what tech? I added flux pinning to be available for einstein's sheid. Also working on multiple supplementary angles, just gotta make an equation for it.


Sent from my Apple Watch

**Reply by smallbigman** [reply]
in the screenshot it shows one of them and the other one is flux pinning but I think you can keep that

**Reply by smallbigman** [reply]
and at some point I would lose collision then get it back the losing collision also goes to the orbs and blocks but the orbs just keep going until a certain point

[Screen recording 2024-11-05 7.07.56 PM.webm](https://github.com/user-attachments/assets/eefe5f58-ba22-4767-baa5-41a910f38da9)

**Reply by smallbigman** [reply]
btw this is what happens when you zoom in too much
[Screen recording 2024-11-05 7.40.31 PM.webm](https://github.com/user-attachments/assets/213bf592-8a53-440f-bdbc-d50dafc688a0)

**Reply by smallbigman** [reply]
also when you have supplementary angles and explosive hollow-points the explosive hollow points only works on the main bullet

**Reply by tbxyd** [reply]
what an interesting glitch (the zoom one). for the collision glitch, I've had that happen to me in vanilla n-gon recently, so nothing on the mods part makes it happen. and I just forgot to remove meissner effect after I finished testing einstein's shield, it doesn't do anything if you choose it

**Reply by smallbigman** [reply]
k thanks the collision glitch stopped and do you think maybe you could fix the supplementary glitch?

**Reply by tbxyd** [reply]
yeah, I'll work on that later (its also not a glitch, just haven't implemented multiple yet). currently mainly working on new sniper tech, such as one coming out today 👀 also new link for game: c-rxxp-y.github.io/n-gon-enhanced

**Comment by AnnonymousNerd87** [comment]
You should have tactical efficiency stack, maybe with diminishing returns.

Also some tech that increases bullet speed would be nice because the low
bullet speed has been the hardest part of using the sniper gun.

**Comment by smallbigman** [comment]
love the new tech but do you think you could make people testers by letting us go to testing mode?

**Comment by tbxyd** [comment]
Like beta testers?


Sent from my Apple Watch

**Reply by smallbigman** [reply]
ye

**Comment by tbxyd** [comment]
Idk, that'd be kinda hard. The way i mod is by programming in vscode on a master copy of the game, then basically copy/paste all that i changed into the repo. I would have to create a whole seperate step in pushing new updates and patches, plus my mods are barebones at the moment so anyone who plays and gives feedback kinda is a tester. What did you have in mind though?


Sent from my Apple Watch

**Reply by smallbigman** [reply]
basically someone who just has testing mode unlocked

**Reply by smallbigman** [reply]
for example
![Screenshot 2024-11-06 6 34 03 PM](https://github.com/user-attachments/assets/15e23ed0-0436-486f-abd6-ca761f2da3c4)

**Reply by tbxyd** [reply]
lets move our discussion to here: https://github.com/c-rxxp-y/n-gon-enhanced/discussions/2

**Comment by tbxyd** [comment]
Also just pushed new update, check readme. Fixed some damage glitches, reworked supplementary angle, and general bug fixes


Sent from my Apple Watch

**Comment by tbxyd** [comment]
Oh anyone can have it unlocked, its built into the game. To get it kill the final boss after getting 5 undefined tech (could be more or less based on difficulty)


Sent from my Apple Watch

**Comment by tbxyd** [comment]
Plus i can't just give specific people testing mode, its a public browser game based in js, so you could do it yourself in the browser console


Sent from my Apple Watch

**Reply by 454077** [reply]
Provided, of course, that if you're using a managed device, your admin hasn't blocked developer tools
<img width="254" height="145" alt="No Dev Tools :(" src="https://github.com/user-attachments/assets/e7e10f17-2dd9-4907-a8d9-a6769c409f91" />

**Comment by btcool** [comment]
I have a mod(?) I made. It is more like a mod loader, since I was fed up of clicking on 5 bookmarklets to play the game.
It's contained in a single file(js/mod.js) and can thus be integrated with any other mod without major conflict.
[Here](https://btcool.github.io/n-gon/)
I'd love to have it implemented into the list.

**Reply by cubingpenguin** [reply]
what mods does it combine

**Reply by btcool** [reply]
It shows all the mods on the mod setting page, and credits the authors in the about section.

**Comment by 454077** [comment]
Here's my [repository](https://github.com/454077/n-mod) if you want to see how I've done it.

**Comment by creator1116** [comment]
its kinda obsolete now that landgreen implemented something like this in the base game
